dind/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+
+    cluster1:
+        image: anybox/clusterdind
+        #restart: unless-stopped
+        ports:
+            - "3501:3500"
+        volumes:
+            - /mnt/cluster1:/var/lib/buttervolume
```

Comment:
I'm wondering if it would be cleaner to mount a local directory instead /mnt but in any case we require sudo access to mount this so it does'nt really matter if we can do it on travis that's ok !